recipes/regenie/conda_build_config.yaml

```diff
@@ -0,0 +1,9 @@
+fortran_compiler_version:
+  - 7
+  # - 9
+boost:
+  - 1.63
+  - 1.65
+  - 1.70
+  - 1.72
+  - 1.74
```

Comment:
Do you really needs builds for so many additional boost versions?

Reply:
At least in the beginning, trying to migrate a bunch of unrelated packages up from boost 1.55 actually

Reply:
OK

Reply:
thanks for the build config update, I was missing fortran 9 version for one before, but now it should hopefully pass